src/DataFormat.ts

```diff
@@ -11,6 +11,6 @@ const DataFormat = {
     SB3: 'sb3',
     SVG: 'svg',
     WAV: 'wav'
-};
+} as const;
 
-module.exports = DataFormat;
+export type DataFormat = typeof DataFormat[keyof typeof DataFormat];
```

Comment:
What does `typeof DataFormat[keyof typeof DataFormat]` do? Is that a common TS idiom?

Reply:
Ah, yes, it's basically saying "give me the type of all possible values that DataFormat can have". So that would be the type `'"jpg" | "json" | "mp3" | "png" | "sb2" | "sb3" | "svg" | "wav"`. We can also say `string`, but since they are an enumeration I wanted to be more specific just in case somewhere this pays off.

In  `typeof DataFormat[keyof typeof DataFormat]`:
- `typeof DataFormat` is `{JPG: 'jpg', JSON: 'json', ...}`
- `keyof {JPG: 'jpg', JSON: 'json', ...}` is `"JPG" | "JSON" | ...` - basically the type of all possible keys as a union
- `{JPG: 'jpg', JSON: 'json'}['JPG']` is `'jpg'` - like getting the value of a property, but for types
- And you can have unions as the indexing: `{JPG: 'jpg', JSON: 'json', PNG: 'png'}['JPG' | 'PNG']` gets you `'jpg' | 'png'` (basically following all versions of the union)